src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Main {
+    public static void main(String[]args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine [] machines = new Machine[] {truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            doWork(machine);
+            stopWork(machine);
+        }
+    }
+
+    public static void doWork(Machine machine) {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/195922337-3541bb43-8140-4408-8af7-d51522a186ef.png)
You don't need this methods in Main class.